player.tl

```diff
@@ -35,6 +35,9 @@ global record Player
 	jump_g: Vector
 	jump_target: Point
 	jump_visual_target: Point
+	-- Using any due to circular dependency: bombs.tl requires player.tl
+	-- Teal doesn't seem to support type-only imports
+	bomb_beneath: any
```

Comment:
This was annoying to leave here.. Do you have any suggestions on how we can dodge this perhaps?